package.json

```diff
@@ -8,13 +8,14 @@
     "preinstall": "./scripts/check_gm.sh",
     "shrinkwrap": "npmshrink",
     "start": "scripts/run_dev.js",
-    "test": "grunt test --node-env=test"
+    "test": "USE_REDIS=false CACHE_EXPIRES_IN=2000 grunt test --node-env=test"
   },
   "dependencies": {
     "aws-sdk": "2.4.7",
     "bluebird": "3.4.1",
     "boom": "4.0.0",
     "buf": "0.1.0",
+    "catbox-memory": "2.0.4",
     "catbox-redis": "3.0.1",
```

Comment:
we would need to update npm shrinkwrap to include these `npm run shrinkwrap` via npm@2